filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -177,6 +179,10 @@ OpenGLDriver::OpenGLDriver(OpenGLPlatform* platform, const Platform::DriverConfi
     slog.i << "OS version: " << mPlatform.getOSVersion() << io::endl;
 #endif
 
+    // Timer queries are core in GL 3.3, otherwise we need EXT_disjoint_timer_query
+    // iOS headers don't define GL_EXT_disjoint_timer_query, so make aboslutely sure
```

Comment:
```suggestion
    // iOS headers don't define GL_EXT_disjoint_timer_query, so make absolutely sure
```

Reply:
thank you ben. I will fix in a follow-up PR.